.gitignore

```diff
@@ -6,6 +6,7 @@
 # Folders
 _obj
 _test
+.idea
```

Comment:
Can we remove this? This should belong to your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

Reply:
Yes, my fault...